__tests__/login/oidc/OidcLoginHandler.spec.ts

```diff
@@ -55,8 +52,9 @@ describe("OidcLoginHandler", () => {
     const actualHandler = defaultMocks.oidcHandler;
     const handler = getInitialisedHandler({ oidcHandler: actualHandler });
     await handler.handle({
+      sessionId: "mySession",
       oidcIssuer: new URL("https://arbitrary.url"),
-      redirect: new URL("https://app.com/redirect"),
+      redirectUrl: new URL("https://app.com/redirect"),
```

Comment:
Seems suspicious - i.e. no '.toString()' here - can in other places below too...???

Reply:
This is currently the interface that leads to this code: 
```typescript
export default interface ILoginOptions {
  sessionId: string;
  oidcIssuer?: URL;
  redirectUrl?: URL;
  clientId?: string;
  clientSecret?: string;
  clientName?: string;
  popUp?: boolean;
  handleRedirect?: (redirectUrl: string) => unknown;
}
```

Both `oidcIssuer` and `redirectUrl` are typed as `URL`, and not `string`. There is some URL manipulation going on in the code (e.g. checking for query params) which is based on this. We could however change the type to `string` and not expect a `URL`, but that feels like a use case different enough to `solid-client-core` that we could use `URL` here and not there.  I'm also happy to align.

Reply:
I don't have too much context here, but I keep emphasising that the "IRI's" in solid-client are actually Named Nodes - which this is not.

Reply:
Fixed in ab0a102d14793ea02b4a99a62ec835f32d1eba80

Reply:
I'm not sure what you mean here @Vinnl - in this case the type should be URL (as in the 'normal' JS URL class), and not a string. I don't think this has anything to do with RDF at all, and therefore nothing to do with Named Nodes.
As a **_completely separate_** point of discussion though, we should chat about the difference between URLs and Named Nodes (I chatted with Nic about this a while ago too). But basically, in RDF, there (generally!) is no distinction whatsoever, e.g. https://bbc.co.uk can most definitely be **_considered_** a Named Node, even if it never returns anything to do with RDF. In RDF, anything that has **_any_** form of IRI (e.g. URI, URL, IRI, URN, DID, etc.) can be considered a Named Node. But anyway, a chat for another time perhaps...